build.jam

```diff
@@ -0,0 +1,29 @@
+# Copyright René Ferdinand Rivera Morell 2023-2024
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE_1_0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+require-b2 5.2 ;
+
+import feature ;
+
+feature.feature boost.nowide.lfs : auto no : optional propagated ;
+
+constant boost_dependencies :
+    /boost/config//boost_config
+    /boost/filesystem//boost_filesystem ;
```

Comment:
This requirement isn't really correct and is causing the failure on Appveyor.
This is only referenced in an optional header provided for convenience and hence not a hard requirement and used only [in a test](https://github.com/boostorg/nowide/pull/183/files#diff-1f3dccb3928ad7b52e4c986270cb619b663197283b764c2fb68256d285f21a3fR37) where it is included with werror disable as Boost.FileSystem isn't warning free.

Can this be removed? If so I'd leave the semicolon on a separate line for smaller future diffs as in other instances
```suggestion
    ;
```